test/js/node/http/node-http.test.ts

```diff
@@ -1079,3 +1081,38 @@ describe("server.address should be valid IP", () => {
     });
   });
 });
+
+it("should not accept untrusted certificates", async () => {
```

Comment:
can you add more tests and copy some of node's tests? they should cover non-node:http usage too. Can you also add tests for WebSocket cert verification?

Reply:
Added, also waiting tests for run